dallinger/docker/ssh_templates/docker-compose-experiment.yml.j2

```diff
@@ -16,10 +16,12 @@ services:
     depends_on: &commondepends
       redis_{{ experiment_id }}:
         condition: service_healthy
+      pgbouncer:
+        condition: service_started
```

Comment:
@silviot I suppose if we're nervous about intermittent errors coming up in the future we could make pgbouncer an opt-out feature using config.txt? So that if people are having problems they can revert back to the original implementation by setting `pgbouncer = false`?